src/stores/api/types/stores.go

```diff
@@ -0,0 +1,22 @@
+package types
+
+import "time"
+
+// TODO: Keep this type as an example of the request type when a manifest is read
+type CreateHashicorpStoreRequest struct {
```

Comment:
Since this is not use and not decided yet it would be used at some point, I would propose to remove it

Reply:
ah yes forgot to remove